applications/tari_merge_mining_proxy/src/main.rs

```diff
@@ -515,12 +515,10 @@ fn main() {
             e
         })
         .unwrap();
-    let _ = stream_handler(listener, transient, rules, rt)
-        .map_err(|e| {
-            error!(target: LOG_TARGET, "{}", e);
-            e
-        })
-        .unwrap();
+    let _ = stream_handler(listener, transient, rules, rt).map_err(|e| {
+        error!(target: LOG_TARGET, "{}", e);
+        e
+    });
```

Comment:
Surprised clippy didn't complain 
```suggestion
    if let Err(e) = stream_handler(listener, transient, rules, rt) {
        error!(target: LOG_TARGET, "{}", e);
    }
```